core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/SearchResourceRequest.java

```diff
@@ -62,7 +62,11 @@
 		/**
 		 * Sort result by the specified sort fields.
 		 */
-		SORT_BY;
+		SORT_BY,
+		/**
+		 * Search by the specific tooling id.
+		 */
+		TOOLING_ID;
```

Comment:
This is not the right place for this `OptionKey`, since this enum is abstract and available for all `SearchResourceRequest` implementations. Please move this to a new `OptionKey` into the `CodeSystemSearchRequest` class.

Reply:
Done